doc/styles/main.css

```diff
@@ -29,49 +29,84 @@ hr {
   border-bottom: 1px solid #ddd;
 }
 
+h1,h2,h3,h4 {
+  padding-top: 10px !important;
+}
+
 /* Top nav bar border bottom */
 .layout-container > header {
+  position: fixed;
   background-color: #333;
-  border-bottom: none;
+  box-sizing: border-box;
+  border-bottom: 0;
+}
+
+.layout-container > header:before {
+  content: 'RxJS';
+  background-image: url('/manual/asset/Rx_Logo_S.png');
+  width: 150px;
+  height: 40px;
+  color: #EC0C8E;
+  font-size: 20px;
+  background-size: auto 25px;
+  background-repeat: no-repeat;
+  padding-left: 30px;
+  padding-right: 5px;
 }
 
 .layout-container > header a {
   color: #777;
 }
 .layout-container > header a:hover, .layout-container > header a:active {
-  color: #fff;
+  color: #FFF;
 }
 
 .layout-container > header > a.repo-url-github {
-  -webkit-filter: brightness(20);
-  filter: brightness(20);
-}
-
-.search-box img {
-  -webkit-filter: brightness(1.7);
-  filter: brightness(1.7);
+  -webkit-filter: brightness(10);
+  filter: brightness(10);
```

Comment:
Brightness 10 makes the github logo look rather white-pinkish, which is never something associated with the github logo and also we don't have light pink colors anywhere else in the page, so I'd suggest we revert this to brightness 20.